src/ZEO/asyncio/client.py

```diff
@@ -888,20 +891,27 @@ def close(self):
                 raise self.exception
 
 class Fut(object):
-    """Lightweight future that calls it's callback immediately rather than soon
+    """Lightweight future that calls it's callbacks immediately rather than soon
     """
 
+    cbv = None
```

Comment:
Isn't `Fut()` usually followed by a call to `add_done_callback` ?

Then I'd replace with:
```py
def __init__(self):
    self.cbv = []
```
remove the `if self.cbv is None:` clause in `add_done_callback`, and the `if self.cbv:` lines in `set_*`.

Reply:
Good idea. Thanks for the advice.